addon/package.json

```diff
@@ -16,7 +16,7 @@
       "title": "backend URL",
       "description": "URL of the backend service to store pages",
       "type": "string",
-      "value": "https://pageshot.dev.mozaws.net"
+      "value": "https://pageshotfzzzy.dev.mozaws.net/"
```

Comment:
Also related to this, Benson did a successful deploy of this branch yesterday on his infrastructure, and noticed that this value needs to come from the environment, not be hard coded.